core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/impl/SimpleEvaluationStrategy.java

```diff
@@ -154,7 +155,7 @@
  * @author David Huynh
  * @author Andreas Schwarte
  */
-public class SimpleEvaluationStrategy implements EvaluationStrategy, UUIDable {
+public class SimpleEvaluationStrategy implements EvaluationStrategy, FederatedServiceResolverClient, UUIDable {
```

Comment:
Can this change be separated out into a separate pull request? It doesn't look like it is related to the rest of this one.